### ISSUE TITLE
Reactivity doesn't seem to work?

### ISSUE THREAD

**Issue by satyavh** [maximummeteor/server-transform#1]
I was really looking for a solution like this package, but so far reactivity doesn't work.
Following your docs, I've setup this serverTransform

```
Collections.serverTransform(
  postCounter: (collection) ->
    Posts.find(
      collections: collection._id
    , 
      reactive: true
    ).count()
)
```

and then I publish the collection using

```
Meteor.publishTransformed "collections", ->
  Collections.find()
```

it returns the count correctly the first time. but if I make new Posts the counter is not updated. So it seems it lacks reactivity.

What am I doing wrong?

**Comment by maxnowack** [comment]
It's fixed in 0.3.5
Thank you for reporting this issue! :)

**Comment by satyavh** [comment]
Thanks, works great now!
